governance.rst

```diff
@@ -0,0 +1,75 @@
+**********
+Governance
+**********
+
+Roles
+=====
+
+Nengo is made by a
+:ref:`team of contributors <People>`
+who review each other's work.
+New contributors join the team
+when they make a pull request
+that gets merged in a project
+managed by the Nengo team.
+
+In addition to being a contributor,
+Nengo team members may
+also be a maintainer and/or a reviewer.
+
+Every Nengo project has a primary maintainer,
+who is responsible for merging pull requests,
+and making releases.
+While the maintainer may delegate maintenance tasks
+to other contributors,
+they are always responsible for
+guiding the project's overall direction.
+
+Nengo contributors can volunteer
+to review a project's pull requests
+by contacting the maintainer of that project.
+
+Policies
+========
+
+Development of all Nengo projects
+happens on `Github <https://github.com/nengo>`_.
+For projects in the Nengo organization,
+we follow these policies.
+
+1. We require that all contributors sign the
+   :ref:`Nengo Contributor Assignment Agreement`.
+
+2. We require all pull requests to be well tested.
+
+3. We follow the
+   `semantic versioning specification <http://semver.org/>`_.
+
+4. We require that every pull request be reviewed
+   by at least one Nengo reviewer.
+
+5. We require large changes to multiple projects
+   be `proposed <https://github.com/nengo/enhancement_proposals>`_
+   prior to implementation.
+
+Projects that have not yet been released
+or have not yet reached the version 1.0.0 milestone
+are under active development;
+the API may change significantly between releases.
+Once a project reaches version 1.0.0,
+we follow these additional policies.
+
+6. We require that every pull request be reviewed
+   by at least two Nengo reviewers.
```

Comment:
We could remove this policy in general and have individual repos (just Nengo at this point) override with additional reviewers if desired. Let's discuss at tomorrow's meeting!